package.json

```diff
@@ -12,6 +12,7 @@
     "format:write": "prettier --write 'src/**/*.ts'",
     "prebuild": "npm run copy:readme",
     "start": "ts-node ./src/bin/svg-to-ts.ts -s ./inputfiles",
+    "start:regex": "ts-node ./src/bin/svg-to-ts.ts -s ./inputfilesRegex/**/*.svg",
```

Comment:
I just ran this command locally and I am confused that commander already adjust this path to a correct file. So inside our svg-to-ts we don't get this path but we get the path to the expressionless file. So instead of getting 
```./inputfilesRegex/**/*.svg``` we get ```./inputfilesRegex/expressionLess/expressionLess.svg```

Reply:
Sorry, I don't understand your point :s

Reply:
I think we need to adjust the startscript in the following way. 

````
    "start": "ts-node ./src/bin/svg-to-ts.ts -s './inputfiles/*.svg'",
````

Reply:
Without the single quotes, this expression is already evaluated as a Regex